renovate.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I added the config file to `.github/renovate.json`

I will update this branch once #444 is merged. I think RenovateBot needs to run regularly in order to respond to PR changes.